operations/mimir-mixin/dashboards/compactor.libsonnet

```diff
@@ -235,6 +235,18 @@ local fixTargetsForTransformations(panel, refIds) = panel {
     )
     .addRow(
       $.row('')
+      .addPanel(
+        $.timeseriesPanel('Estimated Compaction Jobs') +
+        $.queryPanel('sum(cortex_bucket_index_estimated_compaction_jobs{%s}) and (sum(rate(cortex_bucket_index_estimated_compaction_jobs_errors_total{%s}[$__rate_interval])) == 0)' %
+                     [$.jobMatcher($._config.job_names.compactor), $.jobMatcher($._config.job_names.compactor)], 'Jobs') +
+        $.panelDescription(
+          'Estimated Compaction Jobs',
+          |||
+            Estimated number of compaction jobs based on latest version of bucket index. Ingesters upload new blocks every 2 hours (shortly after 01:00 UTC, 03:00 UTC, 05:00 UTC, etc.),
+            and compactors should process all of them within 2h interval. If this graph regularly goes to zero (or close to zero) in 2 hour intervals, then compaction works as designed.
```

Comment:
is there any "gotchas" around the estimation? My understanding is that it's only delayed, but in general the number there was true at some point in the past, right?

Reply:
> My understanding is that it's only delayed, but in general the number there was true at some point in the past, right?

* Yes, "delay" or bucket-index not being 100% up-to-date is one reason.
* Additionally, when computing this number, we only consider jobs that can be executed at given moment. But there are typically more jobs -- however they are blocked. For example job for merging blocks in time range 02:00 -- 04:00 is blocked until all "split" jobs for this time range have finished. Only "split" jobs would be counted toward the estimate and those "merge" jobs would not.
* Another is that when this is computed, there may be ongoing compaction jobs (perhap. 

So it's safe to say that computed number was the minimum number of jobs at some recent point in time.

Reply:
i was going to suggest to include this in the panel descriptions so it's easier to know how reliable this is and to also explain things like the split/merge jobs shadowing (finished split jobs actually creating more merge jobs)

Reply:
Sure, I can update the description.

Reply:
Being done here: https://github.com/grafana/mimir/pull/7481